.github/workflows/cont_integration.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           cargo update -p zstd-sys --precise "2.0.8+zstd.1.5.5"
           cargo update -p time --precise "0.3.20"
           cargo update -p home --precise "0.5.5"
+          cargo update -p proptest --precise "1.2.0"
```

Comment:
Why do we need all these pins?

We don't have to test on MSRV we just have to check that it builds on MSRV. dev-dependencies shouldn't have to be pinned. Or at least that's the policy I'd like to adopt.

See sophisticated technology on how to do this: https://github.com/bitcoindevkit/coin-select/blob/990226a982bd6036bed5a7674fe57d6f9d48e49e/.github/workflows/test.yml#L40

cc @notmandatory

Reply:
Maybe in the future PR? I commented on discord as well about this.

Reply:
Ok issue created: https://github.com/bitcoindevkit/bdk/issues/1398